style_links.json

```diff
@@ -263,10 +411,26 @@
     "name": "Submarine Animation",
     "link": "Submarine Animation/index.html"
   },
+  {
+	"name": "Text Effect",
+	"link": "text effect\index.html"
+  },
```

Comment:
i think the slash is incorrect

Reply:
Well spotted. I used windows and changed the slashes manually. Next commit coming about now